qcfractal/qcfractal/components/neb/record_socket.py

```diff
@@ -400,6 +400,7 @@ def submit_singlepoints(
                 opt_spec = neb_orm.specification.optimization_specification.model_dict()
                 qc_spec = opt_spec["qc_specification"]
             qc_spec["driver"] = "hessian"
+            service_state.iteration = -1
```

Comment:
This is my only concern. This could require a database migration, but I think you are the only one using NEB for now. So it might be safe to ignore that

Reply:
Let me find a better way that can avoid a database migration.